File: src/main/java/no/nav/pensjon/regler/domain/kode/JustertMinstePensjonsnivaTypeCti.java
package no.nav.pensjon.regler.domain.kode;

public class JustertMinstePensjonsnivaTypeCti extends TypeCti {

    private static final long serialVersionUID = 2562762958299802189L;

    public JustertMinstePensjonsnivaTypeCti() {
        super();
    }

    public JustertMinstePensjonsnivaTypeCti(String kode) {
        super(kode);
    }

    public JustertMinstePensjonsnivaTypeCti(JustertMinstePensjonsnivaTypeCti fremskrevetMPNTypeCti) {
        super(fremskrevetMPNTypeCti);
    }

}

File: src/main/kotlin/no/nav/pensjon/regler/domain/kode/JustertMinstePensjonsnivaTypeCti.kt
package no.nav.pensjon.regler.domain.kode

@Deprecated("Ikke i bruk")
class JustertMinstePensjonsnivaTypeCti : TypeCti {
    constructor() : super()
    constructor(kode: String?) : super(kode)
    constructor(fremskrevetMPNTypeCti: JustertMinstePensjonsnivaTypeCti) : super(fremskrevetMPNTypeCti)
}